frontend/src/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../../css/flexbox";
+@import "../../../css/font-awesome/font-awesome.scss";
+@import url('https://fonts.googleapis.com/css?family=Roboto:100,100i,200,200i,300,300i,400,400i,500,500i');
+
+.DirtyBorder {
+  border: 4px solid #40B3FF
+}
+
+.CrossingListItemContainer {
+  width: 1280px;
```

Comment:
Figure out how to move away from hardcoded width here. Need to groom out logic for responsive UI.